### ISSUE TITLE
Windows installer is ignoring installation path

### ISSUE THREAD

**Issue by shana** [git-lfs/git-lfs#819]
I just ran the windows installer and it had the Desktop as the default path for the installation (which isn't really the best place to put executables, tbh).

I changed the path, and it extracted the uninstaller to the custom path I gave it, but the `git-lfs.exe` executable went to the Destkop anyway and not the custom path.

**Comment by technoweenie** [comment]
Doh. IIRC, nsis forced me to add an [install dir](https://github.com/github/git-lfs/blob/b0318b25350b9abfe56af0afb01913a9b2ed56a4/script/nsis/installer.nsi#L4), though it [looks for Git in common locations](https://github.com/github/git-lfs/blob/b0318b25350b9abfe56af0afb01913a9b2ed56a4/script/nsis/installer.nsi#L21-L34). I was hoping to get around having to edit the system path.

I wanted to try out InnoSetup for a windows installer in the next release. Hopefully it's better than this funky nsis scripting language.

**Comment by shana** [comment]
The easiest way to make an installer is to use Wix, really. That's what I use for the visual studio extension, there's no funky/weird UIs or things, it's just command line tools and a simple template file to  create an installer, most straightforward thing ever.

**Comment by shana** [comment]
I can throw a wix template together quickly if you want to try it out. What are the rules for the installer? (i.e., what does it have to put where, are there registry keys, are there arch-specific things it needs to do, etc)

**Comment by technoweenie** [comment]
> I can throw a wix template together quickly if you want to try it out.

That would be amazing :metal: A cli interface is definitely preferable when cutting releases. Especially if I have multiple builds to deal with. I started with nsis because I found a `makensis` command in homebrew, and I had dreams of building windows installers from my Mac. Unfortunately, I needed Windows to sign the installer anyway, so I never actually used `makensis`.

> The default installation dir is so the uninstaller (or the original msi) can live somewhere for later repairs/uninstalls, but there's way better places than the desktop.

Yea, agreed.

> What's the lowest supported version of Windows?

Ideally, whatever the lowest supported version for Git For Windows.
